management/views/product.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse_lazy
+from django.views.generic import (
+    DetailView,
+    UpdateView, ListView,
+    CreateView, DeleteView
+)
+
+from management.forms import ProductForm
+from management.models import Product
+
+
+class ProductProfileView(DetailView):
+    model = Product
+    template_name = 'product_management/view_product_profile.html'
+
+    def get_context_data(self, **kwargs):
```

Comment:
We can do similar thing using Django Context Manager. We might do that in future.